src/keri/app/cli/commands/multisig/join.py

```diff
@@ -196,7 +201,12 @@ def incept(self, attrs):
 
         if approve:
             if self.auto:
-                alias = "test alias"
+                alias = self.alias
```

Comment:
This does not account for auto being true and alias being None which will cause an exception below. (As happened in the scripts run for this PR).

Reply:
This was changed in the most recent commit to have a nested if statement checking of the alias (now "group") is set.
For example:
```python
            if self.auto:
                if self.group is None:
                    group = "default-group"
                else:
                    group = self.group
```